mkdocs.yml

```diff
@@ -23,7 +23,7 @@ extra:
     - type: 'globe'
       link: https://safe-stack.github.io/
 
-pages:
+nav:
```

Comment:
What's this change for? For me, moving to `nav` gives a warning. Maybe we have different versions of mkdocs?

Reply:
OK - I had an old version of mkdocs, I've updated now - all is good.